apps/researcher/src/app/[locale]/objects/[id]/metadata.tsx

```diff
@@ -173,14 +173,19 @@ export function AddMetadataEnrichment({enrichmentType, translationKey}: Props) {
       <div className="flex justify-end text-consortiumBlue-800">
         <SlideOutButton
           id={`${enrichmentType}-form`}
-          className="py-2 px-3 transition flex items-center gap-1 p-1 sm:py-2 sm:px-3 rounded-full text-xs bg-consortiumGreen-300 hover:bg-consortiumGreen-200 text-consortiumBlue-800"
+          className="py-2 px-3 p-1 sm:py-2 sm:px-3 rounded-full text-xs bg-consortiumGreen-300 text-consortiumBlue-800 transition flex items-center gap-1 hover:bg-consortiumGreen-200 whitespace-pre-wrap"
         >
-          <>
-            {t.rich('addUserEnrichmentButton', {
-              name: () => <span>{t(translationKey)}</span>,
-            })}
-            <ChatBubbleBottomCenterTextIcon className="w-4 h-4 fill-consortiumBlue-800" />
-          </>
+          <ChatBubbleBottomCenterTextIcon className="w-4 h-4 stroke-consortiumBlue-800" />
+          <div className="flex flex-col items-start">
```

Comment:
I would prefer to use one text for this button. I think it will work with:

```
        <SlideOutButton
          id={`${enrichmentType}-form`}
          className="py-2 px-3 p-1 sm:py-2 sm:px-3 rounded-full text-xs bg-consortiumGreen-300 text-consortiumBlue-800 transition flex items-center gap-1 hover:bg-consortiumGreen-200"
        >
          <ChatBubbleBottomCenterTextIcon className="w-4 h-4 stroke-consortiumBlue-800" />
          <div className="whitespace-pre-wrap text-left leading-[.5rem]">
            {t.rich('addUserEnrichmentButtonLine', {
              name: () => (
                <span className="lowercase">{t(translationKey)}</span>
              ),
            })}
          </div>
        </SlideOutButton>
```